### DISCUSSION TITLE
DHI.Mike1D support .NET 5.0?

### DISCUSSION THREAD

**Discussion by hawklorry** [DHI/Mike1D-SDK-examples#23] [Q&A] [answered]
It seems DHI.Mike1D doesn't support .NET 5.0. There are warnings after added to the project as shown below. Is there a version that supports .NET 5.0? 

![image](https://user-images.githubusercontent.com/3594975/171056411-e8243cfc-3722-4f4b-9f92-dc024eef9ac6.png)

**Comment by JesperGr** [comment, marked as answer]
As of 2022: If you only need to read result files, then the components for that is .NET Standard 2.0, and hence compatible with .NET 5.0. The rest, especially the computational engine, is .NET Framework 4.7.2. We are working on making the engine independent of .NET Framework in the upcoming release, ready in end of 2022.

**Reply by hawklorry** [reply]
Thanks. I will stick to .NET Framework for now.
